examples/python/notebook/requirements.txt

```diff
@@ -1,3 +1,4 @@
 ipython<=8.12 # ipython 8.13 or greater doesn't work with python 3.8
+tornado<=6.1 # later versions don't work with recent jupyter releases...
```

Comment:
Please note which version doesn't work. If it is 6.2 that is broken now, maybe it is time to re-asses when `tornado` 7.0 is released, for instance